packages/nimble-components/src/anchor/tests/anchor.mdx

```diff
@@ -0,0 +1,24 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import TargetDocs from '../../patterns/anchor/tests/target-docs.mdx';
+import * as anchorStories from './anchor.stories';
+
+<Meta of={anchorStories} />
+
+<Title of={anchorStories} />
+<Description of={anchorStories} />
+
+<DocsStory of={anchorStories.anchor} expanded={false} />
+<Controls of={anchorStories.anchor} />
+
+# Usage Docs
+
+## Accessibility
```

Comment:
Not sure why, but this is rendering weird:
![image](https://github.com/ni/nimble/assets/7282195/eafa79cf-8b21-4780-afe3-3b01ad7fd0a4)

Reply:
Ah, I see the problem for this component specifically. There are styles in the anchor story that are affecting these docs. I'll fix that.